rdbms/types/operations.cpp

```diff
@@ -6,72 +6,90 @@
 #include <algorithm>
 #include <iostream>
 #include <set>
+#include<string>
 
-table operations::runion(table R1 ,table R2){
-    //conditional statment will not work tuple comparing function must be implimented
-    if (!tuples::isEqual(R1.get_attributes() , R2.get_attributes())){
```

Comment:
we need this conditional....

Reply:
without that condition, this code will display an empty set if every element inside are equal

Reply:
But what if the tables have different schemas? this time it'll crush or missbehave

Reply:
I took care of that on the interface unless they have the same relational schema the union won't be called

Reply:
but that is not the write place.....because it's a requirement for the operation not for the interface....if you want to implement new other interface then you don't have to worry on handling of such errors

Reply:
ok I will create another branch and will fix it, for now, merge it